src/pudl/glue/ferc1_eia.py

```diff
@@ -326,17 +326,17 @@ def label_plants_eia(pudl_out: pudl.output.pudltabl.PudlTabl):
         .sum(min_count=1)
         .pipe(apply_pudl_dtypes, group="eia")
     )
-    most_recent_date = max(plants.report_date)
     plants_w_capacity = (
         plants.merge(
             plant_capacity,
             on=["plant_id_eia", "report_date"],
-            how="left",
+            how="outer",
             validate="1:1",
         )
         .assign(link_to_ferc1=lambda x: x.capacity_mw >= MIN_PLANT_CAPACITY_MW)
+        .sort_values(["report_date"], ascending=False)
         .loc[
-            (plants.report_date == most_recent_date),
+            :,
```

Comment:
I decided to sort by date instead of using the most recent date. if a plant doesn't show up in the last year it was getting missed